src/PowerShellEditorServices.Protocol/LanguageServer/GetCommandsRequest.cs

```diff
@@ -0,0 +1,27 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.PowerShell.EditorServices.Protocol.MessageProtocol;
+using System.Collections.Generic;
+
+namespace Microsoft.PowerShell.EditorServices.Protocol.LanguageServer
+{
+    public class GetCommandsRequest
+    {
+        public static readonly
+            RequestType<List<PSCommandMessage>, object, object, object> Type =
+            RequestType<List<PSCommandMessage>, object, object, object>.Create("powerShell/getCommands");
+    }
+
+    public class PSCommandMessage
+    {
+        public string Name { get; set; }
+        public string ModuleName { get; set; }
+        public string DefaultParameterSet { get; set; }
+        public System.Management.Automation.CommandTypes CommandType { get; set; }
```

Comment:
Can you add a `using` for SMA? That way you don't have to repeat yourself.

Reply:
Fixed 😃